test/aes_sky130hs.metrics

```diff
@@ -35,35 +35,35 @@
 	"design__violations": 0,
 	"route__net": 16448,
 	"route__net__special": 2,
-	"route__drc_errors__iter:1": 10210,
-	"route__wirelength__iter:1": 1935891,
-	"route__drc_errors__iter:2": 1444,
-	"route__wirelength__iter:2": 1931073,
-	"route__drc_errors__iter:3": 823,
-	"route__wirelength__iter:3": 1928514,
-	"route__drc_errors__iter:4": 9,
-	"route__wirelength__iter:4": 1928611,
+	"route__drc_errors__iter:1": 10538,
+	"route__wirelength__iter:1": 1935194,
+	"route__drc_errors__iter:2": 1470,
+	"route__wirelength__iter:2": 1930326,
+	"route__drc_errors__iter:3": 892,
+	"route__wirelength__iter:3": 1927887,
+	"route__drc_errors__iter:4": 5,
+	"route__wirelength__iter:4": 1928012,
 	"route__drc_errors__iter:5": 0,
-	"route__wirelength__iter:5": 1928614,
+	"route__wirelength__iter:5": 1928014,
 	"route__drc_errors": 0,
-	"route__wirelength": 1928614,
-	"route__vias": 147988,
-	"route__vias__singlecut": 147988,
+	"route__wirelength": 1928014,
+	"route__vias": 148809,
+	"route__vias__singlecut": 148809,
 	"route__vias__multicut": 0,
 	"DRT::drv": "0",
 	"antenna__violating__nets": 28,
 	"antenna__violating__pins": 29,
 	"DRT::ANT::errors": "28",
 	"timing__drv__floating__nets": 0,
 	"timing__drv__floating__pins": 0,
-	"DRT::worst_slack_min": "-0.10011003420708257",
-	"DRT::worst_slack_max": "-1.1712799950350825",
-	"DRT::tns_max": "-185.9091777588127",
-	"DRT::clock_skew": "0.3285242171289876",
-	"DRT::max_slew_slack": "-0.5167311057448387",
+	"DRT::worst_slack_min": "-0.08416267920804237",
+	"DRT::worst_slack_max": "-1.277694874955022",
+	"DRT::tns_max": "-184.2043725246883",
+	"DRT::clock_skew": "0.3508333722836589",
+	"DRT::max_slew_slack": "-1.185829285532236",
 	"DRT::max_fanout_slack": "100.0",
-	"DRT::max_capacitance_slack": "-2.4792462568314235",
+	"DRT::max_capacitance_slack": "-3.1841214563165265",
 	"DRT::clock_period": "2.811000",
-	"flow__warnings__count": 21,
+	"flow__warnings__count": 41,
```

Comment:
What are the extra warnings?

Reply:
They weren't added because of this update. The master branch results also have 41 warnings. It was probably added at some point, but we didn't update the metrics then.